Packs/SecurityIntelligenceServicesFeed/Integrations/SecurityIntelligenceServicesFeed/SecurityIntelligenceServicesFeed.py

```diff
@@ -313,17 +326,36 @@ def validate_limit(limit: str) -> None:
         raise ValueError(MESSAGES['INVALID_LIMIT_ERROR'])
 
 
+def get_max_indicators(params: Dict) -> int:
+    """
+    Validates the max indicators parameter.
+    If the max_indicators is not a positive integer then throws a ValueError.
+
+    :param params: Demisto parameters.
+    :return: Integer max indicators to fetch in an interval.
+    """
+    try:
+        max_indicators_int = int(params.get('MaxIndicators', ''))
+        if max_indicators_int <= 0:
+            raise ValueError
+        return max_indicators_int
+    except ValueError:
+        raise ValueError(MESSAGES['INVALID_MAX_INDICATORS_ERROR'])
+
+
 def prepare_date_string_for_custom_fields(date_string: str) -> str:
     """
     Prepares date string in iso format and adds timezone if not exist.
 
     :param date_string: string represent date.
     :return: formatted date string.
     """
-    parsed_dt = parse(date_string)
-    if parsed_dt.tzinfo is None or parsed_dt.tzinfo.utcoffset(parsed_dt) is None:
-        parsed_dt = parsed_dt.replace(tzinfo=timezone.utc)
-    return parsed_dt.isoformat()
+    parsed_dt = dateparser.parse(date_string)
+    if parsed_dt:
+        if parsed_dt.tzinfo is None or parsed_dt.tzinfo.utcoffset(parsed_dt) is None:
```

Comment:
What does this code do and why is it necessary after checking if `parsed_dt.tzinfo is None`?
```
parsed_dt.tzinfo.utcoffset(parsed_dt)
```

Reply:
Agree @hod-alpert will remove that redundant check.